apollo-router/Cargo.toml

```diff
@@ -65,7 +65,7 @@ thiserror = "1.0.30"
 tokio = { version = "1.17.0", features = ["full"] }
 tokio-util = { version = "0.7.0", features = ["net", "codec"] }
 tonic = { version = "0.6.2", optional = true }
-tower = { version="0.4.12", features = ["full"] }
+tower = { version = "0.4.12", features = ["util"] }
```

Comment:
We only just changed this to "full". see: #538 It should be full.

Reply:
fixed in def7ea6a